components/AlgoliaSearch.js

```diff
@@ -11,7 +11,13 @@ export const DOCS_INDEX_NAME = 'asyncapi-docs';
 const APP_ID = 'Z621OGRI9Y';
 const API_KEY = '5a4122ae46ce865146d23d3530595d38';
 
-const SearchContext = createContext()
+const SearchContext = createContext({
+  isOpen: false,
+  onOpen: () => {},
+  onClose: () => {},
+  onInput: () => {},
```

Comment:
Why are the functions, part of React Context?

Reply:
done